src/parameters/parameters_flow_solver.cpp

```diff
@@ -94,6 +96,7 @@ void FlowSolverParam::parse_parameters(dealii::ParameterHandler &prm)
         else if (flow_case_type_string == "burgers_viscous_snapshot")  {flow_case_type = burgers_viscous_snapshot;}
         else if (flow_case_type_string == "burgers_rewienski_snapshot")  {flow_case_type = burgers_rewienski_snapshot;}
         else if (flow_case_type_string == "naca0012")  {flow_case_type = naca0012;}
+        else if (flow_case_type_string == "advection_periodic")  {flow_case_type = advection_periodic;}
```

Comment:
add this here as well: 
https://github.com/dougshidong/PHiLiP/blob/1bed06d44bd9bcae1733c92efbca7b313262d81d/src/testing/flow_solver_cases/flow_solver_case_base.cpp#L37-L49

Reply:
Adding in next commit.